projects/addon-doc/components/main/main.template.html

```diff
@@ -11,6 +11,7 @@
         <ng-content select="tuiDocHeader" />
         <button
             appearance="secondary"
+            aria-label="Switch between dark and light mode"
```

Comment:
You can just write this directly inside the button, that's exactly what we have font-size: 0 in icon buttons

Reply:
What do you mean about directly?